src/main/java/com/paneedah/mwc/entities/Entities.java

```diff
@@ -22,14 +22,17 @@ public static void init(CommonProxy commonProxy) {
         new EntityConfiguration.Builder()
                 .withName("terrorist")
                 .withBaseClass(EntityCustomMob.class)
-                .withMaxHealth(40)
+                .withMaxHealth(20)
                 .withEntityIdSupplier(() -> 10000)
                 .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.EASY, 0.1f, Magazines.VSSVintorezMag)
-                .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.HARD, 0.1f, Magazines.ASValMag)
-                .withEquipmentOption(Guns.MakarovPM, EnumDifficulty.EASY, 0.1f, Magazines.MakarovMag)
                 .withEquipmentOption(Guns.Remington870, EnumDifficulty.EASY, 0.07f)
+                .withEquipmentOption(Guns.AK47, EnumDifficulty.NORMAL, 0.07f, Magazines.AK47Mag)
+                .withEquipmentOption(Guns.VSSVintorez, EnumDifficulty.HARD, 0.1f, Magazines.ASValMag)
+                .withEquipmentOption(Guns.HS12, EnumDifficulty.HARD, 0.1f)
+                .withSecondaryEquipmentOption(Guns.Python, EnumDifficulty.NORMAL, 0.1f)
                 .withPrimaryEquipmentDropChance(0.4f)
-                .withSpawn(10, 0, 15, BiomeDictionary.Type.PLAINS, BiomeDictionary.Type.HILLS)
+                .withMaxAmmo(1000)
```

Comment:
AI already has max ammunition, by default is it 36000

Reply:
> AI already has max ammunition, by default is it 36000

Which is vitrually infinite since they shoot about once every second.
Anyway irrelevant, I did not make this, but they should have infinite ammunition, they shouldn't check against a max ammount, they should just shoot if they desire so.